pkgs/applications/networking/mailreaders/mswatch/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchzip
+, pkg-config
+, autoreconfHook
+, bison
+, flex
+, glib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mswatch";
+  # Stable release won't compile successfully
+  version = "unstable-2018-11-21";
+
+  src = fetchzip {
+    url = "https://sourceforge.net/code-snapshots/svn/m/ms/mswatch/code/mswatch-code-r369-trunk.zip";
+    hash = "sha256-czwwhchTizfgVmeknQGLijYgaFSP/45pD2yhDKj5BKw=";
+  };
+  nativeBuildInputs = [
+    pkg-config
+    autoreconfHook
+    bison # For yacc
+    flex
+  ];
+  buildInputs = [
+    glib
+  ];
+
+  meta = with lib; {
+    description = "A command-line Linux utility that efficiently directs mail synchronization between a pair of mailboxes.";
```

Comment:
`meta.description` should not end with a period.

See https://nixos.org/manual/nixpkgs/unstable/#var-meta-description and https://github.com/NixOS/nixpkgs/blob/c097b146a5dba93c473201adc218747d642c2269/CONTRIBUTING.md#submitting-changes